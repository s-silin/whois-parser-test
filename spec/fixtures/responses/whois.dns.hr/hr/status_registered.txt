%
% From May 25, 2018, it  is mandatory  to apply  the  provisions  of the
% General Data Protection Regulation (EU) 2016/679. The basic purpose of
% the regulation is greater control for  individuals over their personal 
% information.
%
% In  accordance  with  the  provisions  of  the General Data Protection 
% Regulation (EU) 2016/679,  we inform you that the personal data of the 
% domain holder and  the contact persons under  the domain holder are no 
% longer publicly available through the domain search engine.
%
% If you believe that there is a legitimate reason for which you need to 
% be provided with the data about the domain  holder please fill  in the 
% Request form for personal data disclosure :
% https://domene.hr/portal/files/Request_personal_data_disclosure.pdf
%

Domain Name: google.hr
Registrar WHOIS Server: whois.dns.hr
Registrar URL: http://whois.dns.hr
Updated Date: 2012-08-28T22:00:00Z
Creation Date: 2011-09-20T22:00:00Z
Registrar Registration Expiration Date: 2018-09-20T22:00:00Z
Registrant Name: Google Hrvatska d.o.o.
Registrant Street: Ulica Petra HektoroviÄ‡a 2
Registrant City: Zagreb
Registrant State/Province: HR
Registrant Postal Code: 10000
Name Server: ns1.google.com
Name Server: ns2.google.com
