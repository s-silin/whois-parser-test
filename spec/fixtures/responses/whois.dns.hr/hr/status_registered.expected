#disclaimer
  %s %ERROR{AttributeNotSupported}


#domain
  %s == "google.hr"

#domain_id
  %s %ERROR{AttributeNotSupported}


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %TIME{2011-09-20T22:00:00Z}

#updated_on
  %s %TIME{2012-08-28T22:00:00Z}

#expires_on
  %s %TIME{2018-09-20T22:00:00Z}

#registrar
  %s %ERROR{AttributeNotSupported}

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}

#admin_contacts
  %s %ERROR{AttributeNotSupported}

#technical_contacts
  %s %ERROR{AttributeNotSupported}

